frontend/src/components/dashboard/geo_location/MapList.vue

```diff
@@ -44,20 +44,30 @@
                 if (!this.dataItems.length) {
                     return [];
                 }
-                let totalSum = 0;
 
-                this.dataItems.forEach(item => {
-                    totalSum += Number(item[this.displayedParameter]) || 0;
-                });
+                if(this.displayedParameter === 'bounce_rate') {
+                    return this.dataItems.map((item) => {
+                        return {
+                            country: item.country,
+                            percentage: Math.round(Number(item[this.displayedParameter])*100)
+                        };
+                    });
+                } else {
+                    let totalSum = 0;
```

Comment:
no need to wrap all this block in else statement, if already has return, so if you unwrap this block nothing will change, but it will be cleaner